models/org.go

```diff
@@ -65,13 +65,11 @@ func (org *User) GetMembers() error {
 		return err
 	}
 
-	org.Members = make([]*User, len(ous))
+	var ids = make([]int64, len(ous))
 	for i, ou := range ous {
-		org.Members[i], err = GetUserByID(ou.Uid)
-		if err != nil {
-			return err
-		}
+		ids[i] = ou.Uid
 	}
+	org.Members, _ = GetUsersByIDs(ids)
```

Comment:
But if get users failed, we should return error to external of function. I think this should be
```Go
org.members, err = GetUsersByIDs(ids)
return err
```